src/bda/plone/orders/mailnotify.py

```diff
@@ -481,3 +523,11 @@ def notify_booking_cancelled_shopmanager(event):
 
 NOTIFICATIONS['booking_cancelled'].append(notify_booking_cancelled_customer)
 NOTIFICATIONS['booking_cancelled'].append(notify_booking_cancelled_shopmanager)
+
+
+def notify_stock_threshold_reached_shopmanager(event):
+    notify_stock_threshold_reached(event, who="shopmanager")
```

Comment:
we only send to shopmanager at all, and who is interpreted in a useless way in notify_stock_threshold_reached